tasks/79_calculate_primoral_of_a_number/calculate_primoral_of_a_number.js

```diff
@@ -0,0 +1,21 @@
+export function numPrimorial(num) {
+  let i = 2;
+  let result = 1;
+  while (num) {
+    if (isPrime(i)) {
+      result *= i;
+      num--;
+    }
+    i++;
+  }
+  return result;
+}
+
+function isPrime(num) {
```

Comment:
I think you can write this function once at top directory and reuse the one in a few tasks to reduce code duplication.
Feel free do it or not